bitnami/grafana-operator/Chart.yaml

```
@@ -24,4 +24,4 @@ name: grafana-operator
 sources:
   - https://github.com/grafana-operator/grafana-operator
   - https://github.com/bitnami/bitnami-docker-grafana-operator
-version: 2.0.4
+version: 2.0.5
```

Could you change this to 2.1.0?
```suggestion
version: 2.1.0
```